http/swagger.yml

```diff
@@ -6557,6 +6552,9 @@ components:
             labels:
               $ref: "#/components/schemas/Link"
       required: [id, name, orgID, flux]
+    TaskStatusType:
```

Comment:
We have a domain typed status called influxdb.Status. I think you can just call it Status.